src/model/paste/__tests__/DraftPasteProcessor-test.js

```diff
@@ -130,8 +130,8 @@ describe('DraftPasteProcessor', function() {
   });
 
```

Comment:
Updated 'p' to become 'div' since we were using 'p' with the assumption that would default to unstyled